src/components/header/index.tsx

```diff
@@ -28,16 +49,33 @@ export const Header = () => {
 
         <Separator size="small" horizontal />
 
-        <StyledLinkButton>
-          <img src={logar} alt="logar" />
-          <LinkButtonComponent color="baseGray">Entrar</LinkButtonComponent>
+        {isMobile && (
+          <StyledMenuHeader>
+            <img src={menu} alt="menu" />
+          </StyledMenuHeader>
+        )}
+
+        {!isMobile && (
+          <StyledLinkButton>
+            <img src={logar} alt="logar" />
+            <LinkButtonComponent color="baseGray" size="large" weight="semiBold">
+              Entrar
+            </LinkButtonComponent>
+          </StyledLinkButton>
+        )}
+
+        <StyledCartIcon onClick={addToCart}>
+          <img src={shopping} alt="carrinho de compras" className="carrinhoCompras" />
```

Comment:
```suggestion
          <img src={shopping} alt="imagem de carrinho de compras" className="cartShopping" />
```
Usar nomes em inglês para classes

Reply:
ajeitei